pkg/chain/ethereum/ethutil/adapter_test.go

```diff
@@ -192,9 +159,10 @@ func TestEthlikeAdapter_PendingNonceAt(t *testing.T) {
 type mockAdaptedEthereumClient struct {
 	*mockEthereumClient
 
-	blocks       []*big.Int
-	transactions map[common.Hash]*types.Receipt
-	nonces       map[common.Address]uint64
+	blocks        []*big.Int
+	blocksBaseFee []*big.Int
+	receipt       *types.Receipt
```

Comment:
Regarding `receipt`, please see: https://github.com/keep-network/keep-common/pull/84/commits/d4355e98. Regarding `blocksBaseFee`, I think it should stay here as base fee is an integral part of blocks since London.